apps/core/src/mvc/mvc.module.ts

```diff
@@ -1,8 +1,17 @@
 import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+
 import { MvcController } from './mvc.controller';
+import { authConfig } from '../config/auth.config';
+import { edgeConfig } from '../config/edge.config';
+import { globalConfig } from '../config/global.config';
 
 @Module({
-  imports: [],
+  imports: [
+    ConfigModule.forFeature(authConfig),
+    ConfigModule.forFeature(edgeConfig),
+    ConfigModule.forFeature(globalConfig),
```

Comment:
Nice catch, was something previously broken for globalConfig and authConfig since it wasn't added here?

Reply:
Atm, it shouldn't be breaking anything since those config modules are defined under app module as well
Just wanted to add them to keep track of the dependencies in case we want to refactor